packages/v4/patternfly-docs/content/design-guidelines/components/skeleton/skeleton.md

```diff
@@ -3,8 +3,6 @@ id: Skeleton
 section: components
 ---
 
-A **skeleton** is a type of loading state that allows you to expose content incrementally. For content that may take a long time to load, use a [progress bar](/components/progress/react) in place of a skeleton.
```

Comment:
progress bar link is dead because of the extra "react" at the end of the link

Reply:
Thanks for calling out the fix needed!